### PR TITLE
Bradleykh

### PR DESCRIPTION
**Pull request by BradleyKH**

### CODE REVIEW

File: LoggingKata.Test/TacoParserTests.cs [hunk 1]
```diff
@@ -31,6 +31,7 @@ public void ShouldParse(string str)
         [Theory]
         [InlineData(null)]
         [InlineData("")]
+        [InlineData("hickory, dickory, dock")]
```
Comment (bdowds) [inline LoggingKata.Test/TacoParserTests.cs]: What if I passed in more than 3 values separated by commas?

Comment (bdowds) [inline LoggingKata.Test/TacoParserTests.cs:36]: What if I passed in more than 3 values separated by commas, or values that are not legitimate longitudes or latitudes?

File: LoggingKata/TacoParser.cs [hunk 2]
```diff
@@ -9,10 +11,50 @@ public class TacoParser
...
+                return null;
+            }
+
+            // If the array.Length is greater than 3, something went wrong
+            if (cells.Length > 3)
+            {
+                logger.LogWarning("The array is too long!");
+                return null;
+            }
+
```
Comment (bdowds) [inline LoggingKata/TacoParser.cs:32]: What if the longitude and latitude values passed in are not legitimate values?

**Review by bdowds — CHANGES_REQUESTED** [review]
Overall Good Work!

File: LoggingKata/Program.cs [hunk 3]
```diff
@@ -15,15 +15,44 @@ static void Main(string[] args)
...
 
-            // TODO:  Find the two Taco Bells in Alabama that are the furthest from one another.
-            // HINT:  You'll need two nested forloops
+            Console.WriteLine("\nLOCATION 1");
+            Console.WriteLine("  Name: " + tbell1.Name);
+            Console.WriteLine("  Point: " + tbell1.Location.ToString());
+            Console.WriteLine("\nLOCATION 2");
+            Console.WriteLine("  Name: " + tbell2.Name);
+            Console.WriteLine("  Point: " + tbell2.Location.ToString());
+
```
Comment (chipelmer) [inline LoggingKata/Program.cs:55]: Small Bonus Challenge: Can you display the max distance formatted in a user-friendly way?

**Pull request merged**